.github/workflows/WebPageToMarkdown.yml

```
@@ -3,26 +3,29 @@ on:
   issues:
     types:
       - opened
+env:
+  issue_body: ${{ github.event.issue.body }}
+  issue_title: ${{ github.event.issue.title }}
 jobs:
   WebPageToMarkdown:
     runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      contents: write
     steps:
-      - name: Does the issue need to be converted to markdown
+      - name: Issue does not need to generate markdown
+        if: ${{ !startsWith(env.issue_title, '[Auto]') }}
         shell: bash
-        run: |
-          if [ "${{ github.event.issue.body }}" ]; then
-            if [[ "${{ github.event.issue.title }}" =~ ^\[Auto\]* ]]; then
-              :
-            else
-              echo "This issue does not need to generate a markdown file." 1>&2
-              exit 1;
-            fi;
-          else
-            echo "The description of the issue is empty." 1>&2
-            exit 1;
-          fi;
+        run: echo "This issue does not need to generate a markdown file." 1>&2; exit 1;
+      - name: The description of the issue is empty
+        if: ${{ !env.issue_body }}
+        shell: bash
+        run: echo "The description of the issue is empty." 1>&2; exit 1;
+
       - uses: actions/checkout@v3
+        if: ${{ env.issue_body && startsWith(env.issue_title, '[Auto]') }}
```

```suggestion
        if: ${{ startsWith(env.issue_title, '[Auto]') }}
```
issue 内容不存在前面就直接退出了，不需要在后面判断了。